### ISSUE TITLE
Examples in README don't seem to work

### ISSUE THREAD

**Issue by wearhere** [mrackwitz/Version#52]
The README suggests that I ought to be able to do

```swift
let version : Version = UIDevice.currentDevice().systemVersion
```

but when I do this, I get the following error: "Cannot convert value of type 'String' to specified type 'Version'".

Googling around, it [seems like](https://forums.developer.apple.com/thread/112870) the code shouldn't be expected to work?

> `ExpressibleByStringLiteral`… is only involved when dealing with literals… you’re dealing with a string value.

Indeed, when I use a literal instead, like

```swift
let version : Version = "13.4"
```

then it does work.

**Comment by regexident** [comment]
I just released v0.8.0 with an updated README and several cleanups/modernizations.

**Comment by wearhere** [comment]
You're welcome! Thanks for the quick response and for a great library.
